### ISSUE TITLE
ResourceType definition cannot be parsed from external resource

### ISSUE THREAD

**Issue by sichvoge** [mulesoft/api-workbench#97]
Creating the following RAML inside the api workbench:

```yaml
#%RAML 0.8
title: My API


schemas:
  - comic: !include http://gist.githubusercontent.com/y0an/f31fa6449771d69b8d28/raw/3ae3112d9a4f3969d2d84c7c0c642f4ee96e54c2/comic-schema.json
resourceTypes:
  - base: !include http://gist.githubusercontent.com/y0an/3a089190b27596bed143/raw/7adea5ea4ac1ae70febb38e1cce69aa98bb90c40/base.yaml

/test:
  displayName: test
```

I am getting the following error:

![image](https://cloud.githubusercontent.com/assets/498114/12265862/60eb25f2-b91e-11e5-84e9-13ea73feed5f.png)

Copy&Paste the content into a local file and changing the `!include` to point to it works fine. Seems it has problems with the external resource. BTW, the `comic` schema just works fine.

**Comment by sichvoge** [comment]
Maybe the reason is that it is trying to validate the file, but its unknown to the parser as it is a "0.8 fragment"?

**Comment by y0an** [comment]
i don't think because with base.yaml in local directory there is no error.

**Comment by ddenisenko** [comment]
Fix published in 0.8.16.
